doc/changelogs/CHANGELOG_V6.md

```diff
@@ -40,6 +41,170 @@
 [Node.js Long Term Support Plan](https://github.com/nodejs/LTS) and
 will be supported actively until April 2018 and maintained until April 2019.
 
+<a id="6.9.2"></a>
+## 2016-12-06, Version 6.9.2 'Boron' (LTS), @thealphanerd
+
+This LTS release comes with 144 commits. This includes 47 that are docs related, 46 that are test related, 15 which are build / tools related, and 9 commits which are updates to dependencies
```

Comment:
Is this auto-generated? It’s a long line (ditto for v4.7.0) and should probably be ended with a full stop